infra/nnfw/cmake/options/options_armv7l-linux.cmake

```diff
@@ -4,7 +4,9 @@
 option(DOWNLOAD_NEON2SSE "Download NEON2SSE library source" OFF)
 option(BUILD_OPENCL_TOOL "Build OpenCL tool" ON)
 
+option(DOWNLOAD_OPENGL_HEADERS "Download Opengl_headers source" ON)
+option(DOWNLOAD_EGL_HEADERS "Download Egl_headers source" ON)
+option(DOWNLOAD_VULKAN "Download vulkan source" ON)
```

Comment:
why are theses needed for linux?

Reply:
These are needed for building tensorflow lite. So I added it.

Reply:
As I understand these are options and these can be swiched on from the ourside.
Settings these ON is these are necessary for the build and turning it off may break the build.

Are these options necessary for OneRT build for linux?
If not please turn it off and switch ON from outside in your build environment.

Reply:
@seanshpark I have checked again the gpu_cl backend history about FLAG in [infra/nnfw/cmake/options/options_armv7l-linux.cmake](https://github.com/Samsung/ONE/pull/9256/files/9c58d6b68222d3c968cbe84fe39b48d04d24fa00#diff-88596ec4ef6912b5d8f9d29c5465de1ec40528a85e1db095415d208e6f3a7b0b)

1. nnfw_api test(unit test) about gpu_cl backend is running on odroid-xu4 in CI server. Thus, the following option should be on default

```
option(BUILD_GPU_CL "Build gpu_cl backend" ON)
option(BUILD_TENSORFLOW_LITE_GPU "Build TensorFlow Lite GPU delegate from the downloaded source" ON)
```

Thus, the patch below might be necessary for the build with tensorflow 2.7 source : 

```
option(DOWNLOAD_OPENGL_HEADERS "Download Opengl_headers source" ON)
option(DOWNLOAD_EGL_HEADERS "Download Egl_headers source" ON)
option(DOWNLOAD_VULKAN "Download vulkan source" ON)
```

IMHO, to update into tf2.7 with gpu_gl, this option is mandatory.

Sorry for confusing.

Reply:
@YiHyunJin Have you seen this error log ? 

```
ubuntu@ubuntu ~> Product/out/bin/nnpackage_run mobilenet_v1_1.0_224_quant/
Package Filename mobilenet_v1_1.0_224_quant/
Product/out/bin/nnpackage_run: symbol lookup error: /home/ubuntu/Product/out/bin/../lib/../lib/../lib/libbackend_gpu_cl.so: undefined symbol: _ZN4absl12lts_2021110212UnknownErrorENS0_11string_viewE
```

I have seen this on rpi4 armv7l ubuntu. ( note that rpi does not provide OpenCL DDK)